cmd/client/wallet/main.go

```diff
@@ -537,6 +545,48 @@ func blsRecoveryCommand() {
 	}
 }
 
+func importBls() {
+	if err := blsImportCommand.Parse(os.Args[2:]); err != nil {
+		fmt.Println(ctxerror.New("failed to parse flags").WithCause(err))
+		return
+	}
+
+	if *blsKey != "" {
+		privateKey := &ffi_bls.SecretKey{}
+		err := privateKey.DeserializeHexStr(*blsKey)
+		if err != nil {
+			fmt.Printf("Your raw private key is not valid.\n Err: %v\n", err)
+			os.Exit(101)
+		}
+		passphrase := utils.AskForPassphrase("Passphrase: ")
+		passphrase2 := utils.AskForPassphrase("Passphrase again: ")
+		if passphrase != passphrase2 {
+			fmt.Printf("Passphrase doesn't match. Please try again!\n")
+			os.Exit(3)
+		}
+
+		fileName, err := blsgen.WritePriKeyWithPassPhrase(privateKey, passphrase)
+		if err != nil {
+			fmt.Printf("Can not generate encrypted bls file.\n Err: %v\n", err)
+			os.Exit(101)
+		} else {
+			fmt.Printf("Your encrypted bls file with the passphrased is written at %s.\n", fileName)
+		}
+		privateKey2, err := blsgen.LoadBlsKeyWithPassPhrase(fileName, passphrase)
+		if err != nil {
+			fmt.Printf("Error when loading the private key with the passphrase. Err: %v", err)
+			os.Exit(101)
+		}
+		if !privateKey.IsEqual(privateKey2) {
+			fmt.Println("The loaded private key does not match your private key.")
+			os.Exit(101)
+		}
+		fmt.Println("We loaded the file and the loaded private key DOES match your private key!")
+	} else {
+		fmt.Println("The private key should not be empty.")
```

Comment:
I can see the user going “Huh?  Empty?  Did I specify some empty thing?  I didn't?”  Let's frame from the user's perspective:

```suggestion
		fmt.Println("Please specify the hexadecimal private key string using --key")
```